cds_sorenson/api.py

```diff
@@ -27,31 +27,54 @@
 from __future__ import absolute_import, print_function
 
 import json
+from collections import OrderedDict
+from itertools import chain
 
 import requests
 from flask import current_app
 
-from .error import SorensonError
+from .error import InvalidAspectRatioError, InvalidResolutionError, \
+    SorensonError
 from .utils import generate_json_for_encoding, get_status
 
 
-def start_encoding(input_file, preset_ID, output_file=None):
+SORENSON_STATUSES = {
```

Comment:
That's a good question. In my opinion it doesn't (as this is something that is used only in this one place), so we could leave it (and change to statuses, to please PEP-8, although I don't like this convention and I would leave the uppercase).
On the other hand, if we move it to config, people can overwrite the error messages if the want it (dunno if it's useful to anybody, but it would be there just in case).
WDYT?